Aptoide/aptoide-client-v8
# [ASV-1948] Events for app bundles

**What does this PR do?**

   This PR aims at adding an appbundle flag to all the download and install related events.
I also took the chance to add the appc flag to the missing events (it was not being sent in the EditorsChoice ones). This final step was added after talking with Ana.

**Database changed?**

 No

**Where should the reviewer start?**

- [ ] InstallAnalytics.java

**How should this be manually tested?**

Open Aptoide, do a download and install the app. Check that all events were correctly sent. Notice that some of them are sent to BI.
Also confirm that the structure is fine.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1948](https://aptoide.atlassian.net/browse/ASV-1948)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
